mycroft/skills/core.py

```diff
@@ -240,9 +247,17 @@ def initialize(self):
         """
         raise Exception("Initialize not implemented for skill: " + self.name)
 
+    def converse(self, transcript, lang="en-us"):
```

Comment:
shouldn't `transcript` be `utterances` to keep it in line with the wording that has been used previously?